click_repl/__init__.py

```diff
@@ -135,6 +140,13 @@ def repl(old_ctx, prompt_kwargs=None):
     group = group_ctx.command
     isatty = sys.stdin.isatty()
 
+    # Delete the REPL command from those available, as we don't want to allow
+    # nesting REPLs (note: pass `None` to `pop` as we don't want to error if
+    # REPL command already not present for some reason).
+    repl_command_name = old_ctx.command.name
+    available_commands = group_ctx.command.commands
+    available_commands.pop(repl_command_name, None)
```

Comment:
If we do that won't the behaviour be slightly different as you will still be able to run a REPL within a REPL, the command name will just not be autocompleted?